### ISSUE TITLE
[limpieza][P3] Rama muerta en `CerebroBrooder.logits_argumento`

### ISSUE THREAD

**Issue by Justo-Tapiador** [Justo-Tapiador/ia-so-brooder#7]
## Contexto

`brooder/cerebro.py`, en `logits_argumento()`:

```python
if g.dim() == 2:
    entrada = torch.cat([g, embebido], dim=-1)
else:
    entrada = torch.cat([g, embebido], dim=-1)   # idéntico
```

Las dos ramas del `if/else` son exactamente iguales: `torch.cat` con
`dim=-1` funciona igual para tensores 2D y 3D. El condicional es
código muerto que sugiere una diferencia que no existe (y confunde al
lector: "¿qué cambia entre lotes y secuencias?").

## Cambio propuesto

```python
def logits_argumento(self, g, primitivas):
    """Logits del argumento condicionados a la primitiva elegida.

    g: [B, G] o [B, T, G] | primitivas: [B] o [B, T] (long)
    """
    embebido = self.embebido_primitiva(primitivas)
    return self.cabeza_argumento(torch.cat([g, embebido], dim=-1))
```

## Criterios de aceptación

- [ ] El condicional desaparece sin cambio de comportamiento
      (los tests de `test_cerebro.py` cubren ambos casos de forma).
- [ ] `pytest -q` sigue en verde.
